bagua/torch_api/env.py

```diff
@@ -3,62 +3,78 @@
 
 def get_world_size():
     """
-    Returns the number of processes in the current process group
+    Gets the number of processes in the current process group.
```

Comment:
Get. Similar modification to other doc strings.

Reply:
done